### DISCUSSION TITLE
HDFSLineageDispatcher to write to custom hdfs location

### DISCUSSION THREAD

**Discussion by kamalakannan-87** [AbsaOSS/spline-spark-agent#841] [General]
Will you be able to enhance HDFSLineageDispatcher to emit events to specified hdfs location ?

**Comment by wajda** [comment]
Sure. If you can create a PR that would be awesome.

**Reply by kamalakannan-87** [reply]
Sure, Scala 2.11 is not supported in newer release ?

**Reply by kamalakannan-87** [reply]
@wajda Can you please comment on this

**Reply by cerveada** [reply]
It is, we release bundles for each Spark version and not all of them support Scala 2.11.
For Spark versions that support Scala 2.11 we release 2.11 bundles.
